config/configuration.toml.in

```diff
@@ -4,6 +4,8 @@
 path = "@QEMUPATH@"
 kernel = "@KERNELPATH@"
 image = "@IMAGEPATH@"
+# Uncomment if having trouble running pre-2.15 glibc
```

Comment:
I forgot to say, please can you add a comment stating the format of this variable. For example, it could become something like this:

```
# Optional space-separated list of options to pass to the guest kernel.
# For example, uncomment the line below if you are having trouble running pre-2.15 glibc
## kernel_params = "vsyscall=emulate"
```